docs/reference/cluster_manifest.md

```diff
@@ -199,6 +199,9 @@ explanation of `ttl` and `loop_wait` parameters.
   automatically created by Patroni for cluster members and permanent replication
   slots. Optional.
 
+* **standby_cluster**
```

Comment:
just **standby** to align with manifests and Postgresql_type. Also in util_tests.